src/api/app/helpers/webui/package_helper.rb

```diff
@@ -72,4 +72,13 @@ def nbsp(text)
     # But at this point we know it truly is html safe
     result.html_safe
   end
+
+  def humanize_time(secs)
+    [[60, :s], [60, :m], [24, :h], [1000, :d]].map do |count, name|
+      if secs > 0
+        secs, n = secs.divmod(count)
+        "#{n.to_i}#{name}"
+      end
+    end.compact.reverse.join(' ')
```

Comment:
Are you sure this can not be done with some existing rails helpers, like [Numeric](http://api.rubyonrails.org/classes/Numeric.html)?